### DISCUSSION TITLE
Connectors to CASE tools and offline metadata sources

### DISCUSSION THREAD

**Discussion by rk186009** [open-metadata/OpenMetadata#5431] [Ideas]
I think it would be useful to have connectors to other types of metadata sources:
1. CASE tools like ERWin, Power Designer
2. offline sources of metadata: Excel sheets, XML, JSON documents (These would have to be in specific structure.)

These are primarily to cater to new project implementations in an organization where new metadata is being on-boarded or generated.

Does this align with the overall vision of this project? What is the process to discuss this further and subsequently get this requirement added to the backlog/list?

**Comment by harshach** [comment]
@rk186009 yes definitely. But bit lower on the priority. Do you want create a feature request.
